requirements-dev.txt

```diff
@@ -6,6 +6,8 @@ ckantoolkit>=0.0.4
 factory-boy==2.12.0
 flake8==3.8.3
 mock
+pika>=1.1.0,<1.3.0
```

Comment:
we really need to do a requirements-dev-py2.txt, requirments-dev-py3.txt sooner rather than later for these things.

Reply:
Eh, in our own infrastructure project I don't think we need to support multiple versions.

The latest upstream ckanext-archiver requirements handle both Py2 and Py3, once we merge them.